cmd/plasmacli/subcmd/query/height.go

```diff
@@ -0,0 +1,31 @@
+package query
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/spf13/cobra"
+)
+
+// HeightCmd returns the current block height of the plasmad connection
+func HeightCmd() *cobra.Command {
+	return heightcmd
+}
+
+var heightcmd = &cobra.Command {
+	Use:   "height",
+	Short: "Check current block height",
+	Long: "Returns current block height of plasmad connection",
+	SilenceUsage: true,
```

Comment:
have all these files used `gofmt` I feel like the spacing is off

Reply:
just fixed